docs/conf/portland/2018/news/volunteers-shirts-tickets.rst

```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/{{templatecode}}/generic.html
+
+.. post:: March 19, 2018
+   :tags: portland-2018, volunteers, shirts, tickets
+
+Call for Volunteers, Shirts on Sale, Tickets Update
+===================================================
+
+Greetings, fellow documentarians! The Portland conference is just short of two months away, and we can't wait to see you all soon! Here are the latest updates from the conference team to help you prepare.
+
+Call for volunteers
+-------------------
+
+There is plenty to do during the conference and the success of the conference relies on our volunteers. The conference organizers are looking for folks who are willing to help out during the conference. If you volunteer for two or more shifts (clocking in at an average of three hours each), you will receive free registration for the conference.
+
+We are looking for folks to help out in these roles:
+
+- Ballroom setup - Prepping the main conference hall for the main conference, including laying out cables/power strips, arranging tables/chairs, and other physical labor
```

Comment:
*light* physical labour?

Reply:
Define “light” - perhaps carrying boxes or other loads up to ## pounds, kneeling, standing for long periods, etc.?

Reply:
@yo-bj feel free to propose a tweak to the text to reflect "light", I'm not sure what's considered light but you know relatively well what it all means :)